Data_Analysis_Markdown.Rmd

```diff
@@ -0,0 +1,546 @@
+---
+output: html_document
+---
+--- 
```

Comment:
I looked at the diffs (https://github.com/soc504-s2015-princeton/RepProj/pull/3/files) and the changes that you made seem good.